ppb/systems.py

```diff
@@ -191,6 +202,28 @@ def register_renderable(self, renderable):
         source_path = renderable.__resource_path__()
         self.register(source_path / image_name, image_name)
 
+    def resize_image(self, image, game_unit_size):
+        # TODO: Pygame specific code To be abstracted somehow.
```

Comment:
This entire class is pygame-specific code, considering that the only other implementation is pyglet, which does things 100% differently.